Packet++/src/GtpLayer.cpp

```diff
@@ -155,522 +155,521 @@ GtpV1Layer::GtpExtension GtpV1Layer::GtpExtension::createGtpExtension(uint8_t* d
+	tempMap[53] = "Forward Relocation Request";
+	tempMap[54] = "Forward Relocation Response";
+	tempMap[55] = "Forward Relocation Complete";
+	tempMap[56] = "Relocation Cancel Request";
+	tempMap[57] = "Relocation Cancel Response";
+	tempMap[58] = "Forward SRNS Context";
+	tempMap[59] = "Forward Relocation Complete Acknowledge";
+	tempMap[60] = "Forward SRNS Context Acknowledge";
+	tempMap[61] = "UE Registration Request";
+	tempMap[62] = "UE Registration Response";
+	tempMap[70] = "RAN Information Relay";
+	tempMap[96] = "MBMS Notification Request";
+	tempMap[97] = "MBMS Notification Response";
+	tempMap[98] = "MBMS Notification Reject Request";
+	tempMap[99] = "MBMS Notification Reject Response";
+	tempMap[100] = "Create MBMS Notification Request";
+	tempMap[101] = "Create MBMS Notification Response";
+	tempMap[102] = "Update MBMS Notification Request";
+	tempMap[103] = "Update MBMS Notification Response";
+	tempMap[104] = "Delete MBMS Notification Request";
+	tempMap[105] = "Delete MBMS Notification Response";
+	tempMap[112] = "MBMS Registration Request";
+	tempMap[113] = "MBMS Registration Response";
+	tempMap[114] = "MBMS De-Registration Request";
+	tempMap[115] = "MBMS De-Registration Response";
+	tempMap[116] = "MBMS Session Start Request";
+	tempMap[117] = "MBMS Session Start Response";
+	tempMap[118] = "MBMS Session Stop Request";
+	tempMap[119] = "MBMS Session Stop Response";
+	tempMap[120] = "MBMS Session Update Request";
+	tempMap[121] = "MBMS Session Update Response";
+	tempMap[128] = "MS Info Change Request";
+	tempMap[129] = "MS Info Change Response";
+	tempMap[240] = "Data Record Transfer Request";
+	tempMap[241] = "Data Record Transfer Response";
+	tempMap[254] = "End Marker";
+	tempMap[255] = "G-PDU";
 
 	return tempMap;
 }
 
 const std::map<uint8_t, std::string> GTPv1MsgTypeToStringMap = createGtpV1MessageTypeToStringMap();
 
-std::string GtpV1Layer::getMessageTypeAsString()
+std::string GtpV1Layer::getMessageTypeAsString() const
 {
-    gtpv1_header* header = getHeader();
+	gtpv1_header *header = getHeader();
 
-    if (header == NULL)
-    {
-        return GTPv1MsgTypeToStringMap.find(0)->second;
-    }
+	if (header == NULL)
+	{
+		return GTPv1MsgTypeToStringMap.find(0)->second;
+	}
 
-    std::map<uint8_t, std::string>::const_iterator iter = GTPv1MsgTypeToStringMap.find(header->messageType);
-    if (iter != GTPv1MsgTypeToStringMap.end())
-    {
-        return iter->second;
-    }
-    else
-    {
-        return GTPv1MsgTypeToStringMap.find(0)->second;
-    }
+	std::map<uint8_t, std::string>::const_iterator iter = GTPv1MsgTypeToStringMap.find(header->messageType);
+	if (iter != GTPv1MsgTypeToStringMap.end())
+	{
+		return iter->second;
+	}
+	else
+	{
+		return GTPv1MsgTypeToStringMap.find(0)->second;
+	}
 }
 
-bool GtpV1Layer::isGTPUMessage()
+bool GtpV1Layer::isGTPUMessage() const
 {
-    gtpv1_header* header = getHeader();
-    if (header == NULL)
-    {
-        return false;
-    }
+	gtpv1_header *header = getHeader();
```

Comment:
please move the `*` back to its original place. This is the convention in the rest of the project

Reply:
Done